README.md

```diff
@@ -19,23 +19,23 @@ Stage 2 indicates that the committee expects these features to be developed and
-| [Symbols as WeakMap keys][symbols-weakmap]                                     | Daniel Ehrenberg<br />Richard Button<br />Robin Ricard<br />Leo Balter<br />Rick Waldron<br />Caridy Patiño | Daniel Ehrenberg<br />Richard Button<br />Robin Ricard<br />Leo Balter<br />Rick Waldron<br />Caridy Patiño | :question: | <sub>[June&nbsp;2022][symbols-weakmap-notes]</sub>
+| [Legacy RegExp features in JavaScript][regexp-legacy]                          | Claude Pache                                                            | Mark Miller<br />Claude Pache                                           | [legacy-regexp][tests-regexp-legacy]             | <sub>[May&nbsp;2017][regexp-legacy-notes]</sub>            |
+| [`Atomics.waitAsync`][nonblocking]                                             | Lars Hansen                                                             | Shu-yu Guo<br />Lars Hansen                                             | [Atomics.waitAsync][tests-nonblocking]           | <sub>[December&nbsp;2019][nonblocking-notes]</sub>         |
+| [Import Assertions][import-assertions]                                         | Myles Borins<br />Sven Sauleau<br />Dan Clark<br />Daniel Ehrenberg     | Myles Borins<br />Sven Sauleau<br />Dan Clark<br />Daniel Ehrenberg     | [import-assertions][import-assertions-tests]     | <sub>[November&nbsp;2020][import-assertions-notes]</sub>   |
+| [JSON Modules][json-modules]                                                   | Myles Borins<br />Sven Sauleau<br />Dan Clark<br />Daniel Ehrenberg     | Myles Borins<br />Sven Sauleau<br />Dan Clark<br />Daniel Ehrenberg     | [json-modules][json-modules-tests]               | <sub>[January&nbsp;2021][json-modules-notes]</sub>         |
+| [Temporal][temporal]                                                           | Philipp Dunkel<br />Maggie Johnson-Pint<br />Matt Johnson-Pint<br />Brian Terlson<br />Shane Carr<br />Ujjwal Sharma<br />Philip Chimento<br />Jason Williams<br />Justin Grant | Philipp Dunkel<br />Maggie Johnson-Pint<br />Matt Johnson-Pint<br />Brian Terlson<br />Shane Carr<br />Ujjwal Sharma<br />Philip Chimento<br />Jason Williams<br />Justin Grant| [Temporal][temporal-tests] | <sub>[March&nbsp;2021][temporal-notes]</sub> |
+| [Resizable and growable ArrayBuffers][resizable]                               | Shu-yu Guo                                                              | Shu-yu Guo                                                              | [resizable-arraybuffer][resizable-tests]         | <sub>[May&nbsp;2021][resizable-notes]</sub>                |
+| [ShadowRealm][shadowrealm]                                                     | Caridy Patiño<br />Jean-Francois Paradis                                | Dave Herman<br />Mark Miller<br />Caridy Patiño<br />Leo Balter<br />Rick Waldron<br />Chengzhong Wu | [ShadowRealm][realms-tests] | <sub>[December&nbsp;2021][realms-notes]</sub>      |
```

Comment:
it'd be great to make the feature flag link smaller
```suggestion
| [ShadowRealm][shadowrealm]                                                     | Caridy Patiño<br />Jean-Francois Paradis                                | Dave Herman<br />Mark Miller<br />Caridy Patiño<br />Leo Balter<br />Rick Waldron<br />Chengzhong Wu | <sub>[ShadowRealm][realms-tests]</sub> | <sub>[December&nbsp;2021][realms-notes]</sub>      |
```

Reply:
Done.